pkg/tlsmanage/mgr.go

```diff
@@ -176,6 +185,8 @@ func (m *Manager) SetupRoute53(sess *session.Session, zoneId string) error {
 }
 
 func (m *Manager) SetupHubCert(ctx context.Context) error {
+	m.tlsCertMu.Lock()
+	defer m.tlsCertMu.Unlock()
```

Comment:
I'd move this down to lines 212 or so, after we get the cert.

Reply:
👍 done in 7c396dc